CHANGELOG.md

```diff
@@ -6,12 +6,33 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 ## Breaking Changes
+ - Property scope `Pointer.actorsUnderPointer` changed to private;
 
 ## Added
+ - New `PointerEvent.stopPropagation()` method added. Works the same way as    (https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation)
```

Comment:
Thanks for adding a changelog entry! The changelog file is a place for consumer-facing changes, so these entries can probably be reduced to API changes, and the relevant new/changed features that will be immediately helpful to a user trying to make a game.

In this case, a number of these new methods, etc. can be documented appropriately in the code comments so that consumers can utilize them correctly.

Typically, a completed GitHub issue usually results in a line or two in the changelog, like what you have here at the beginning of the "Added" section.

Reply:
I have removed excessive entries from this Changelog, please, check  if it now looks fine.  :3